### ISSUE TITLE
.status.userEmail is required but isn't set on manually created clusters

### ISSUE THREAD

**Issue by judge-red** [kubermatic/kubermatic#11019]
### What happened?

I'm trying to create a cluster manually (i.e. using kubectl rather than the dashboard wizard). That works well, except that it does not let me set `.status.userEmail` (which makes sense) but also the system doesn't set it either. However, the usercluster-controller crashes without this email being provided:

```
$ k -n cluster-foobar logs usercluster-controller-f7bf694b4-rb4km
Defaulted container "usercluster-controller" out of: usercluster-controller, copy-http-prober (init)
{"level":"info","time":"2022-09-21T10:20:01.386Z","logger":"http-prober","caller":"http-prober/main.go:127","msg":"Probing","attempt":1,"max-attempts":100,"target":"https://apiserver-external.cluster-foobar.svc.cluster.local./healthz"}
{"level":"info","time":"2022-09-21T10:20:01.400Z","logger":"http-prober","caller":"http-prober/main.go:116","msg":"Hostname resolved","hostname":"apiserver-external.cluster-foobar.svc.cluster.local.","address":"10.101.73.175:443"}
{"level":"info","time":"2022-09-21T10:20:01.409Z","logger":"http-prober","caller":"http-prober/main.go:140","msg":"Endpoint is available"}
{"level":"info","time":"2022-09-21T10:20:01.614Z","caller":"cli/hello.go:36","msg":"Starting Kubermatic User-Cluster Controller-Manager (Enterprise Edition)...","version":"v2.20.6"}
{"level":"fatal","time":"2022-09-21T10:20:01.614Z","caller":"user-cluster-controller-manager/main.go:160","msg":"-owner-email must be set"}
```

### Expected behavior

Either `.status.userEmail` is not required or it must be set when manually creating a cluster too

### How to reproduce the issue?

Write a Cluster manifest manually, create the resource and see this field never being set (and it being removed if set in the manifest or the resource being edited manually). Here's my example:

```
apiVersion: kubermatic.k8c.io/v1
kind: Cluster
metadata:
  labels:
    project-id: zg8sh8vz78
  name: foobar
spec:
  cloud:
    dc: openstack-switch-ZH
    openstack:
      credentialsReference:
        name: credential-openstack-hkqrvtbnhk
        namespace: kubermatic
      floatingIPPool: public
      network: ""
      nodePortsAllowedIPRange: 0.0.0.0/0
      routerID: ""
      securityGroups: ""
      subnetID: ""
    providerName: openstack
  clusterNetwork:
    dnsDomain: cluster.local
    konnectivityEnabled: true
    nodeLocalDNSCacheEnabled: true
    pods:
      cidrBlocks:
      - 172.25.0.0/16
    proxyMode: ipvs
    services:
      cidrBlocks:
      - 10.240.16.0/20
  cniPlugin:
    type: canal
    version: v3.22
  containerRuntime: containerd
  enableUserSSHKeyAgent: true
  exposeStrategy: LoadBalancer
  features:
    apiserverNetworkPolicy: true
    ccmClusterName: true
    externalCloudProvider: true
  humanReadableName: expose-strategy-test
  opaIntegration:
    webhookTimeoutSeconds: 10
  pause: false
  version: 1.23.6
status:
  userEmail: noreply@example.com

```

### How is your environment configured?

- KKP version: 2.20.6
- Shared or separate master/seed clusters?: shared

### Provide your KKP manifest here (if applicable)

Doesn't matter, imho

### What cloud provider are you running on?

OpenStack

### What operating system are you running in your user cluster?

Flatcar, but doesn't matter, it never gets that far

### Additional information

I'm trying to create a cluster manually because I want to test the `LoadBalancer` expose strategy while Kubermatic/Seed use the `NodePort` expose strategy.

Note that I've previously opened (and since closed) #11018 as I initially misunderstood the actual issue.

**Comment by embik** [comment]
It seems the issue you are encountering is fixed by #9547, but we can look into backporting this to the 2.20 release series. As a workaround, I think you should be able to add the email address with the [kubectl edit-status](https://github.com/ulucinar/kubectl-edit-status) plugin.

**Comment by embik** [comment]
This will be fixed with the next 2.20 patch release, which will include #11047.

**Comment by embik** [comment]
This has been fixed in [KKP v2.20.8](https://github.com/kubermatic/kubermatic/releases/tag/v2.20.8).
